### PR TITLE
[mini] Fix some of unused variable warnings.

### PR DESCRIPTION
**Pull request by nsivov**

### CODE REVIEW

File: mono/mini/aot-runtime.c [hunk 1]
```diff
@@ -4505,7 +4505,7 @@ init_llvmonly_method (MonoAotModule *amodule, guint32 method_index, MonoMethod *
 	ERROR_DECL (error);
 
 	res = init_method (amodule, method_index, method, init_class, context, error);
-	if (!is_ok (error)) {
+	if (!res) {
```
Comment (vargaz) [inline mono/mini/aot-runtime.c]: This is not the same.

Comment (vargaz) [reply mono/mini/aot-runtime.c]: Guess it is.

**Comment by nsivov** [comment]
@vargaz Ok, done.

**Comment by luhenry** [comment]
@monojenkins build failed

**Pull request merged**